glotaran/io/reader.py

```diff
@@ -36,4 +36,4 @@ def decorator(reader):
     return decorator
 
 
-file_reader(extension='nc', name='netCDF4')(xr.open_dataset)
```

Comment:
Why was this change needed? What's the difference? Is this only applicable to the extra file?
"Open an DataArray from a file or file-like object containing a single data variable.
This is designed to read netCDF files with only one data variable. If multiple variables are present then a ValueError is raised."
Ref:
http://xarray.pydata.org/en/stable/generated/xarray.open_dataset.html
http://xarray.pydata.org/en/stable/generated/xarray.open_dataarray.html

Reply:
Readers must return arrays now, because not everything they read are datasets for fitting. There must be some restrains on the dataset, if people really want to read from multivalue sets and are not able to extract the data to fit as array we can think about supporting datasets at this time.